src/Mermaid/props.ts

```diff
@@ -0,0 +1,139 @@
+import mermaidAPI from 'mermaid/mermaidAPI';
+
+/**
+ * This type contains all the configuration of mermaid to be used as properties of the plugin.
+ * @public
+ */
+export type MermaidProps = {
+  config?: {
```

Comment:
Could we re-use the mermaid [Config type](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/mermaid/mermaidAPI.d.ts#L249) here? This would make maintenance easier going forward.

Reply:
Absolutely, I was looking for something like that in the Mermaid repo and apparently just wasn't looking in the correct place haha